docs/data/routes/docs/nextjs/data-fetching/data-services/en.md

```diff
@@ -13,14 +13,20 @@ JSS Next.js applications support both forms of pre-rendering provided by Next.js
 
 Next.js uses different data fetching strategies for each pre-rendering form. 
 
+>  See the Next.js documentation on [Data Fetching](https://nextjs.org/docs/basic-features/data-fetching) for more information about Next.js data fetching functions, and the `context` argument.
+
 ## JSS Next.js data fetching
 
 The JSS Next.js sample application includes usage examples for both data fetching strategies available in Next.js, as follows: 
 
-* `src/pages/[[path]].tsx` implements `GetStaticPaths` and `GetStaticProps` (SSG).
-* `src/page_examples/[[path]].SSR.tsx` implements `GetServerSideProps` (SSR).
+* [`src/pages/[[..path]].tsx`](https://github.com/Sitecore/jss/blob/master/samples/nextjs/src/pages/%5B%5B%2E%2E%2Epath%5D%5D.tsx) implements `GetStaticPaths` and `GetStaticProps` (SSG).
+* [`src/pages/[[..path]].SSR.tsx`](https://github.com/Sitecore/jss/blob/master/samples/nextjs/src/pages/%5B%5B%2E%2E%2Epath%5D%5D.SSR.tsx) implements `GetServerSideProps` (SSR).
 
->  See the Next.js documentation on [Data Fetching](https://nextjs.org/docs/basic-features/data-fetching) for more information about Next.js data fetching functions, and the `context` argument.
+> You can choose the initial `[[..path]].tsx` pre-rendering form on create with the optional `prerender` parameter. SSG is used by default if you omit the parameter. For example, [with `jss create`](/docs/nextjs/getting-started-nextjs/walkthrough-jsscreate):
+>
+> ```
+> jss create my-first-jss-app nextjs --prerender {SSG|SSR}
+> ```
 
 In the preceding examples, the implementations of `GetStaticProps`  and   `GetServerSideProps` leverage the `SitecorePagePropsFactory`. See the `SitecorePagePropsFactory` definition in [`src/lib/page-props-factory.ts`](https://github.com/Sitecore/jss/blob/master/samples/nextjs/src/lib/page-props-factory.ts).
```

Comment:
Since the page props factory is the entry point for the services, I feel like it should be introduced at the top of this page. Something like this flow:
- we need page props with data about the current context to render a page
- before we do the rendering we create these page props with the factory
- define the page props factory (this would be your current sentence about the factory)
- go into details of what the factory does (this would be the transition to listing the services)

This is probably a decision for @AncaIO

Reply:
Keeping this suggestion in mind for docs refactoring. Good point!